pkg/utils/utils.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+// GetClusterDomainName returns cluster's domain name or error
```

Comment:
In theory for getters go suggests omitting Get, so I'd suggest ClusterDomainName (same below).

Reply:
So Getter/Setter convention in Go is for X is: SetX() and X(), i.e. Get is redundant. 
Considering that here it's not a true getter on a struct, I don't want to be prescriptive, but ClusterDomainName() is shorter, and is readable for Go engineers in general. Up to you :)

Reply:
I am for consistency, if the rules you have stated were followed in serving repo, then it would have made sense to change func name, but if you check, currently `get` is used in many funcs' names in serving:
```go
.//pkg/reconciler/names.go:func GetK8sServiceFullname(name string, namespace string) string {
.//pkg/reconciler/names.go:func GetServingK8SServiceNameForObj(name string) string {
```
and in many more,  as such I prefer to follow this pattern and leave this func name as is.